sdk/monitor/opentelemetry-exporter/src/Declarations/Contracts/Generated/Envelope.ts

```diff
@@ -0,0 +1,55 @@
+// THIS FILE WAS AUTOGENERATED
+import Base = require('./Base');
+'use strict';
+
+/**
+ * System variables for a telemetry item.
+ */
+class Envelope {
+  /**
+   * Envelope version. For internal use only. By assigning this the default, it will not be serialized within the payload unless changed to a value other than #1.
+   */
+  public ver: number;
+
+  /**
+   * Type name of telemetry data item.
+   */
+  public name: string | undefined;
```

Comment:
nit: why not make the field optional? e.g. `public name?:  string`

Reply:
Since the field is actually required by the ingestion server, I don't want intellisense to show certain fields as optional.

Overall, the code-gen for these files is very out of date (not following modern linting rules or latest TS practices). I've had to go and modify the output of them to actually build for latest TS, and the codegen isn't actually run anymore. I think the plan is to use some different codegen at some point (via swagger?), and so I haven't put much effort into improving these.